frontend/src/components/projects/ProjectMeter.vue

```diff
@@ -17,15 +17,23 @@ const { t } = useI18n();
 const meterItems = computed(() => {
     const groups = props.project !== null ? props.project.status.non_empty_groups : 0;
     const groupsSubmitted = props.project !== null ? props.project.status.groups_submitted : 0;
-    const submissionsPassed = props.project !== null ? props.project.status.submissions_passed : 0;
-    const submissionsFailed = groupsSubmitted - submissionsPassed;
+    const structureChecksPassed = props.project !== null ? props.project.status.structure_checks_passed : 0;
+    const extraChecksPassed = props.project !== null ? props.project.status.extra_checks_passed : 0;
+    const submissionsFailed = groupsSubmitted - structureChecksPassed;
+
     return [
         {
-            value: (submissionsPassed / groups) * 100,
+            value: (extraChecksPassed / groups) * 100,

```

Comment:
Isn't it possible to have extrachecks passed but structure checks not? I would change this to the min of those 2.

Reply:
That code not extrachecks and structurechecks for a single submission, but for a whole project (**Poject**Meter).

Reply:
You should round this to the nearest integer to account for representation errors, it could happen that the meter now displays 22.3333333333331%.

Reply:
Fixed

Reply:
Actually, the MeterGroup component already rounds these values by default.